about/about.html

```diff
@@ -11,22 +11,29 @@
 
 <body>
     <header class="landing-page">
-        <div class="home-page">
-            <nav>
-                <div class="nav-bar">
-                    <img class="humbuger" src="./imagesabout/menu.png" alt="Menu" />
-                    <ul class="menu-item-hidden">
-                        <li class="cancel"><img src="./imagesabout/xicon.png" alt=""></li>
-                        <li><a href="../about/about.html" class="about">About</a></li>
-                        <li><a href="#Program" class="nav-list">Program</a></li>
-                        <li><a href="#join" class="nav-list">Join</a></li>
-                        <li><a href="#sponsor" class="nav-list">Sponsor</a></li>
-                        <li><a href="#news" class="nav-list">News</a></li>
-                        <li><a href="#campain" class="campain">CC Campain</a></li>
-                    </ul>
-                </div>
-            </nav>
-
+        <nav>
+        <div class="media">
+            <a href="#" class="twitter"><img src="./imagesabout/twitter.svg" alt="twitter" /></a>
+            <a href="#" class="linkedin"><img src="./imagesabout/linkdin.svg" alt="linkdin" /></a>
+            <a href="#" class="english">English</a>
+            <a href="#" class="page">My Page</a>
+            <a href="#" class="logout">Logout</a>
```

Comment:
- [ ] (OPTIONAL) Kindly make these icons in the dark navbar light so as to follow the template requirements 👍
- Your design:
![Annotation 2022-03-20 060430](https://user-images.githubusercontent.com/85113567/159149529-6992f86a-18b7-4207-ac2a-021eaeae1bb3.jpg)
- Figma design:
![Annotation 2022-03-20 060449](https://user-images.githubusercontent.com/85113567/159149536-2307ea47-d1ad-4b9c-82c2-bba8928b0017.jpg)